packages/shared/src/modules/reviews/libs/types/review-search-query.type.ts

```diff
@@ -0,0 +1,5 @@
+type ReviewSearchQuery = {
```

Comment:
Let's also add an operation here, the search query may differ for different methods
```suggestion
type ReviewGetAllSearchQuery = {
```